index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/1935d064dd.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@100;300;400;500;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100;200;300;400;500;600;700;800;900&family=Roboto:wght@100;300;400;500;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portifolio</title>
+  </head>
+  <body>
+    <header class="header-class">
+      <a href="index.html" id="logo">Filbert</a>
+      <nav id="nav-bar">
+        <i class="fa fa-bars" id="bars"></i>
+      </nav>
+    </header>
+    <main id="main">
+      <section class="headline-container">
+        <h1 class="greatings">Hey There. I'm Filbert.</h1>
+
+        <h2 class="intro-heading">I'am a Software Developer</h2>
+        <p class="intro-content">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hesitate to contact me.
+        </p>

```

Comment:
Nice job using the right tags for user accessibility's sake😉. Let's try to stick to the design by ensuring we use the right font color😉.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="400" alt="image" src="https://github.com/FilbertKimasa/Portfolio/assets/64643995/c368d5c6-7c7e-4de0-a966-aa9e422c9e6a">|<img width="400" alt="image" src="https://github.com/FilbertKimasa/Portfolio/assets/64643995/3d13ff4e-6ed2-4378-8458-7e4da917db03">|